prompts/sources.go

```diff
@@ -42,6 +32,20 @@ func getBaseSourcePrompts(currentWorkflow *workflow.Workflow, sourceName, fileLo
 		)
 	}
 
+	if sourceName == nil || *sourceName == "" {
+		initialGroup = append(initialGroup,
+			charm_internal.NewInput().
+				Title("What is a good name for this source document?").
```

Comment:
I wonder if we could autocomplete better here 🤔

Reply:
Open to any ideas. But I couldn't think of any good defaults.